Update 5_adding_items.rst (#59)
godot-escoria/escoria-docs: getting_started/step_by_step/5_adding_items.rst

parameter and the text to say as the second parameter. (See the
:doc:`documentation </api/SayCommand>`)

> If you don't define an ``ESCLocation`` in your player nodetree, text will appear
> at the top of the screen.
> 
> So, add now an ``ESCLocation`` as a child of ``ESCPlayer``.
> 
> Name this node as "dialog_position" (this name is mandatory).
> 
> By default, text would appear under the character. You can move the node above
> the Graham's head, to get a more regular position.
> 

Reviewer: ```suggestion
If you don't define an ``ESCLocation`` in your player scene, dialog will
appear at the top of the screen.

So add an ``ESCLocation`` as a child of ``ESCPlayer``.

Name this node "dialog_position" (this name is mandatory).

By default, dialog will appear under the character. You can move the node above
Graham's head to have dialog appear in a more regular location.

```
